src/chrome/content/rules/LeMonde.fr.xml

```diff
@@ -0,0 +1,157 @@
+		judo.lemonde.fr
+		legendesmarines.lemonde.fr
+		learn-french.lemonde.fr
+		lecabillaudnorvegien.lemonde.fr
+		lequebecdanstouslessens.lemonde.fr
+		lesartisansdelaperonomie.lemonde.fr
+		lesclesdedemain.lemonde.fr
+		modele-lettre.lemonde.fr
+		mondial.lemonde.fr
+		natation.lemonde.fr
+		prix-immobilier.lemonde.fr
+		progresser-orthographe.lemonde.fr
+		promo.lemonde.fr
+		rugby.lemonde.fr
+		ski.lemonde.fr
+		sport.lemonde.fr
+		sports-us.lemonde.fr
+		support.lemonde.fr
+		tennis.lemonde.fr
+		terresinconnues.lemonde.fr
+		unregardsurlecosse.lemonde.fr
+		voile.lemonde.fr
+		volley.lemonde.fr
+
+	Time out:
+		club.lemonde.fr
+		dev.lemonde.fr
+		jeux.lemonde.fr
+		resultats-examens.lemonde.fr
+		voiture-occasion.lemonde.fr
+
+	Refused:
+		collection-histoire.lemonde.fr
+		comparateurs-banques.lemonde.fr
+
+	Private subdomain:
+		huit.lemonde.fr
+
+	Secure connection failed:
+		paroles2chansons.lemonde.fr
+
+-->
+<ruleset name="LeMonde.fr" platform="mixedcontent">
+
+	<target host="www.lemonde.fr" />
+	<target host="abo.lemonde.fr" />
+	<target host="abonnes.lemonde.fr" />
+	<target host="abosh.lemonde.fr" />
+	<target host="actu.lemonde.fr" />
+	<target host="actu-golf.lemonde.fr" />
+	<target host="athletisme.lemonde.fr" />
+	<target host="auto-moto.lemonde.fr" />
+	<target host="basket.lemonde.fr" />
+	<target host="basket-nba.lemonde.fr" />
+	<target host="blogs.lemonde.fr" />
+	<target host="bourse.lemonde.fr" />
+	<target host="boutique.lemonde.fr" />
+	<target host="campus.lemonde.fr" />
+	<target host="checkout.lemonde.fr" />
+	<target host="concoursasturies.lemonde.fr" />
+	<target host="contact.lemonde.fr" />
+	<target host="cyclisme.lemonde.fr" />
+	<target host="dicocitations.lemonde.fr" />
+	<target host="elles-defient-le-temps.lemonde.fr" />
+	<target host="faq.lemonde.fr" />
+	<target host="finance.lemonde.fr" />
+	<target host="finances.lemonde.fr" />
+	<target host="football.lemonde.fr" />
+	<target host="football-nfl.lemonde.fr" />
+	<target host="formule1.lemonde.fr" />
+	<target host="forums.lemonde.fr" />
+	<target host="golf.lemonde.fr" />
+	<target host="handball.lemonde.fr" />
+	<target host="hockey-nhl.lemonde.fr" />
+	<target host="info.lemonde.fr" />
+	<target host="innovation-sap.lemonde.fr" />
+	<target host="judo.lemonde.fr" />
+	<target host="mondial.lemonde.fr" />
+	<target host="natation.lemonde.fr" />
+	<target host="planetebusiness.lemonde.fr" />
+	<target host="regard-de-pro.lemonde.fr" />
+	<target host="rugby.lemonde.fr" />
+	<target host="secure.lemonde.fr" />
+	<target host="ski.lemonde.fr" />
+	<target host="sport.lemonde.fr" />
+	<target host="sports-us.lemonde.fr" />
+	<target host="tahitivahine.lemonde.fr" />
+	<target host="www.tahitivahine.lemonde.fr" />
+	<target host="tennis.lemonde.fr" />
+	<target host="vins.lemonde.fr" />
+	<target host="voile.lemonde.fr" />
+	<target host="volley.lemonde.fr" />
+
+	<securecookie host="^\.?(?:abo|boutique|monabo)\.lemonde\.fr$" name=".+" />
```

Comment:
`pubs.lemonde.fr` is 410 gone. Do you have a working test URL for this subdomain?